torch/nn/_intrinsic/fake_quantize.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ..modules.module import Module
+from ..._jit_internal import weak_module
+from ...autograd.function import Function
+
+@weak_module
+class DefaultObserverModule(Module):
+    def __init__(self, qconfig, alpha=0.9):
+        super(DefaultObserverModule, self).__init__()
+        self.stats = ()
+        self.alpha = alpha
+
+    def forward(self, X):
+        if not self.stats:
+            self.stats = torch.min(X), torch.max(X)
+        else:
+            self.stats = alpha * self.stats[0] + (1-alpha) * torch.min(X), \
+                alpha * self.stats[1] + (1-alpha) * torch.max(X),
+        return self.stats
+
+def get_default_calcqparam(qconfig):
```

Comment:
Observer module also has a calculate qparam method inside it, lets harmonize with the post training observer code